content/en/docs/tasks/administer-cluster/kubelet-in-userns.md

```diff
@@ -123,29 +123,36 @@ On your node, systemd must already be configured to allow delegation; for more d
 Containers documentation.
 
 ### Configuring network
+
 {{% thirdparty-content %}}
 
 The network namespace of the Node components has to have a non-loopback interface, which can be for example configured with
-slirp4netns, VPNKit, or lxc-user-nic.
+[slirp4netns](https://github.com/rootless-containers/slirp4netns),
+[VPNKit](https://github.com/moby/vpnkit), or
+[lxc-user-nic(1)](https://www.man7.org/linux/man-pages/man1/lxc-user-nic.1.html).
 
 The network namespaces of the Pods can be configured with regular CNI plugins.
 For multi-node networking, Flannel (VXLAN, 8472/UDP) is known to work.
 
 Ports such as the kubelet port (10250/TCP) and `NodePort` service ports have to be exposed from the Node network namespace to
-the host with an external port forwarder, such as RootlessKit, slirp4netns, or socat.
+the host with an external port forwarder, such as RootlessKit, slirp4netns, or
+[socat(1)](https://linux.die.net/man/1/socat).
 
-You can use the port forwarder from K3s; see https://github.com/k3s-io/k3s/blob/v1.21.2+k3s1/pkg/rootlessports/controller.go
+You can use the port forwarder from K3s. See the [documentation](https://rancher.com/docs/k3s/latest/en/advanced/#known-issues-with-rootless-mode)
```

Comment:
nit: Could provide more descriptive link text.

See [Running K3s in Rootless Mode](https://rancher.com/docs/k3s/latest/en/advanced/#known-issues-with-rootless-mode) for more details.
or
For more information, see [Running K3s in Rootless Mode](...).